openassessment/workflow/migrations/0002_auto_20170724_1537.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Update: we're pushing the submissions upgrade through independently, as it had an identical migration. Expectation is that it runs as a no-op, we're going to verify that as submissions goes out since it's a smaller table.